linera-chain/src/manager.rs

```diff
@@ -392,46 +396,48 @@ impl ChainManager {
         outcome: BlockExecutionOutcome,
         key_pair: Option<&KeyPair>,
         local_time: Timestamp,
+        used_blobs: Vec<Blob>,
```

Comment:
Right now this always gets passed over, but will probably most of the time get ignored :/ 
The only other option I could think was passing a `StorageClient` reference over, but then that would mean that `linera-chain` would have to start depending on `linera-storage`, which I'm not sure is something we want...

Reply:
We're conditionally building this now, so we should be good